kv/secret.go

```diff
@@ -120,7 +119,10 @@ func (s *Service) getSecretKeys(ctx context.Context, tx Tx, orgID influxdb.ID) (
 	}
 
 	if id != orgID {
-		return nil, fmt.Errorf("organization has no secret keys")
+		return nil, &influxdb.Error{
```

Comment:
I think we can simply return []string, nil here.

Reply:
Also fixed